S21/indraneelL/wordConvert.py

```diff
@@ -0,0 +1,29 @@
+import locale
+import re
+
+## read the file and iterate through every line
+class WordConvert:
+    def readTestFile(self):
+        textFile = open("tests.tsv", "r")
+        while textFile:
+            line = textFile.readline()
+            if line == "" or line == '\n':
+                break
+            word, lang, _  = line.split('\t')
+            self.methodToProcessText(word,lang)
+        textFile.close()
+
+    def makeWordsLower(self,word):
+        print(word.lower())
+
+    def methodToProcessText(self, text, lang):
+        # using regex check for the language and do functionalities
+        word = text
+        if re.findall("\A[n,t][AEIOUÁÉÍÓÚ]",text) and re.match("^ga.*",lang):
```

Comment:
Good to see you use an array to check the vowels all at once instead of assigning them to variables. Very efficient

Reply:
Thanks for the comment. Really using regular expressions was handy !!